x-pack/solutions/observability/plugins/apm/public/application/index.tsx

```diff
@@ -59,6 +59,8 @@ export const renderApp = ({
     lens: pluginsStart.lens,
     uiActions: pluginsStart.uiActions,
     observabilityAIAssistant: pluginsStart.observabilityAIAssistant,
+    onechat: pluginsStart.onechat,
+    inference: pluginsStart.inference,
```

Comment:
This is the (small) downside of not using the an API in onechat (aka Agent Builder): that we have to supply our own `inference` client.

I'd prefer having everything contained in AB making the implementations between plugins more consistent